coverage-tests.js

```diff
@@ -284,6 +284,40 @@ test('check window fully on android chrome emulator', {
   },
 })
 
+test('check window UFG native', {
+  variants: {
+    'on ios': {
+      env: {
+        device: 'iPhone 12',
+        app: 'https://applitools.jfrog.io/artifactory/Examples/DuckDuckGo-instrumented.app.zip',
+        injectUFGLib: true,
+      },
+      config: {
+        browsersInfo: [
+          {iosDeviceInfo: {deviceName: 'iPhone 12', iosVersion: 'latest'}}
+        ]
+      }
+    },
+    'on android': {
+      env: {
+        device: 'Pixel 3 XL',
+        app: 'https://applitools.jfrog.io/artifactory/Examples/ufg-native-example.apk',
+      },
```

Comment:
Maybe 'injectUFGLib: true,' is needed here the same as it in 'on ios'

Reply:
@YuriiChere no, because the Android app is precompiled with the UFG lib (AFAIK)